.github/workflows/_matrix-and-codecov-on-merge-to-main.yml

```diff
@@ -9,8 +9,8 @@ on:
         required: false
         type: string
       python_versions:
-          description: 'Comma-separated list of Python versions to test within matrix CI, e.g., "3.13, 3.14"'
-          default: "3.12, 3.13, 3.14"
+          description: 'Comma-separated list of Python versions to test within matrix CI, e.g., "3.13, 3.14". If not provided, versions will be read from pyproject.toml'
```

Comment:
Just adding that the `default` value for Python verison parameters is updated by the above resusuable function.